Client/Api/Commands/ITenantIdsCommandStep.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace Zeebe.Client.Api.Commands
+{
+    public interface ITenantIdsCommandStep<out T>
```

Comment:
🔧 We could either add here a single AddTenant or extend the previous introduced interface